themes/mongodb/src/css/mongodb-docs.css

```diff
@@ -704,7 +704,7 @@ div.versionadded > p > span, div.versionchanged > p > span, div.deprecated > p >
 .content .main-column {
     order: 1;
     min-height: 600px;
-    width: 85%;
+    width: 68%;
```

Comment:
It's a mix of both. Any percentage under 69% will work.

The TOC is fixed at 330px, and screen widths over 1093px display the TOC.

1 - 330/1093 = 69.8%.